openpype/hosts/max/plugins/publish/published_files_tycache_object.py

```diff
@@ -0,0 +1,37 @@
+import pyblish.api
+from pymxs import runtime as rt
+
+
+class PublishedFilesTycacheObject(pyblish.api.InstancePlugin):
+    """Load Published Files for Created TyCache Object."""
+
+    order = pyblish.api.IntegratorOrder + 0.89
+    label = "Published files for tycache Object"
```

Comment:
I think the labels/names here can be clearer - what does it do with published files, etc? It feels a bit overly generic naming for what it actually does like "TycacheReplaceLoadedObjectPaths" or whatever.